panel/src/components/Forms/Writer/Writer.vue

```diff
@@ -174,13 +174,13 @@ export default {
 					}
 				},
 				update: (payload) => {
-					if (!this.editor) {
+					if (!payload.editor) {
 						return;
 					}
 
 					// compare documents to avoid minor HTML differences
 					// to cause unwanted updates
-					const jsonNew = JSON.stringify(this.editor.getJSON());
```

Comment:
Just not sure about these two lines. I don't know if there is a difference in the data that `this` and `payload` contain.

Reply:
So the purpose of this change has something to do with code splitting and are we sure it doesn't break anything?

Reply:
In theory we could keep it out of this PR. These are leftovers from my attempt to get Writer split into its own bundle. And there the issue is that relying on `this.editor` being present isn't good, but instead better to work with wha tis passed to the method (as there we can be sure it exists). But maybe indeed easier to just remove the Editor/Writer changes from this PR.

Reply:
Ohh. I can confirm that this change reoccurs the https://github.com/getkirby/kirby/issues/3798 issue. So we should revert this two lines.

Reply:
Let's remove all writer changes from this PR

Reply:
✅